OneCommunityGlobal/HighestGoodNetworkApp
# Jacky fix remove user from task

# Description
Fix: After clicking the “remove user from task” button on the modal, it works but the modal does not close by itself. 


## Related PRS (if any):
N/A

## Main changes explained:
- Remove loading useState in TaskCompleteModal.jsx
- Use react-toastify to notify data has been updated
- Close the modal after confirm removal of user from task

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin/owner user
4. go to dashboard→ Team Member Tasks
5. look for a test user which has task(s) assigned
6. click the red 'x' button on the task under the user to see the changes

## Screenshots or videos of changes:

### Before: 
User can be deleted from the task, but there is no toast shown whether the deletion is successful. The loading modal cannot close itself. 

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/108389005/b379989b-6280-449d-92f0-a41ce0b92732

### After:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/108389005/5d3a3a8d-90cd-4779-87ca-8e2ff0b5492b

## Note:
### How to create task under project and assign user to the task


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/108389005/c5316a1a-38db-4ab5-ac4f-e69d09786974
